python/BUILD.bazel

```diff
@@ -223,6 +224,12 @@ bzl_library(
     ],
 )
 
+bzl_library(
```

Comment:
I thought it would be good to evolve the repo towards a more standardised structure like the one described by [rules-template](https://github.com/bazel-contrib/rules-template)

Alternatively, if we would prefer to paint that bikeshed another time, I can hide as much of the implementation inside `//python/private/pypi` and we can restructure later.